src/sync/incomingSyncUtils.js

```diff
@@ -193,6 +193,14 @@ export const sanityCheckIncomingRecord = (recordType, record) => {
       cannotBeBlank: ['name', 'number_to_use'],
       canBeBlank: [],
     },
+    Pref: {
+      cannotBeBlank: ['ID', 'store_ID', 'item', 'data'],
```

Comment:
was wondering, is there a chance that some global preference would also sync to mobile and cause sync to halt because it doesn't have store_ID ? Maybe it's more stable to check store_ID in `case 'Pref': {`

Reply:
AFAIK, it the sanity check rules are broken the record is just silently ignored (no halting of syncing). Correct me if wrong?

Reply:
From memory it was halting sync, I did a quick check, didn't see any non store specific prefs syncing